### PR TITLE
Update admin slides for 2021 GAT + videos

### PR DESCRIPTION
**Pull request by hexylena**
xref https://github.com/galaxyproject/admin-training/issues/104

### CODE REVIEW

File: topics/contributing/tutorials/slides-with-video/tutorial.md [hunk 1]
```diff
@@ -75,30 +75,31 @@ The captioning process is completely automated, but it means that for very long
...
+
+Word       | Pronunciation
+---------- | ---
+SQLAlchemy | SQL alchemy
+FastQC     | fast QC
+nginx      | engine X
+gxadmin    | GX admin
+/etc       | / E T C
+
+This is the same for the many terms we read them differently than they're written, e.g. 'src' vs 'source'. Most of us would pronounce it like the latter, even though it isn't spelt that way. Our speaking robot doesn't know what we mean, so we need to spell it out properly.
```
Comment (nsoranzo) [inline topics/contributing/tutorials/slides-with-video/tutorial.md:100]: That's great! If we could only have a way to tell it how to discriminate (in context) between the 2 pronunciations of "analyses": the plural of "analysis" vs. the 3rd person singular present tense of "to analyse": https://www.youtube.com/watch?v=o2LRIgxjeP0

Comment (hexylena) [reply topics/contributing/tutorials/slides-with-video/tutorial.md:100]: So in *theory* that should be taken care of by context with the TTS model and not our responsibility! Which is super nice. But there seem to be some edge cases with AWS. E.g. "Many analyses were done" is pronounced correctly, but "together we did multiple analyses" doesn't. So, context matters for the engines, and we may need to rephrase things occasionally for this.

Comment (nsoranzo) [inline topics/contributing/tutorials/slides-with-video/tutorial.md:100]:
```suggestion
The same applies to the many terms we read differently from how they are written, e.g. 'src' vs 'source'. Most of us would pronounce it like the latter, even though it isn't spelt that way. Our speaking robot doesn't know what we mean, so we need to spell it out properly.
```

Comment (hexylena) [reply topics/contributing/tutorials/slides-with-video/tutorial.md:100]: Ah it's wrong I guess in the intro slides then. Ok, I'll see if I can find an OK rephrasing, thanks for alerting on that!

File: topics/admin/tutorials/ansible-galaxy/slides.html [hunk 2]
```diff
@@ -92,3 +155,11 @@
 
 [galaxy-role]: https://galaxy.ansible.com/galaxyproject/galaxy
 [ansible-galaxy-tutorial]: {% link topics/admin/tutorials/ansible-galaxy/tutorial.md %}
+
+???
+
+- Here are the basic best practices.
+- Run without privileges to limit blast radius.
```
[the remark this answers is not in the document]
Comment (hexylena) [reply topics/admin/tutorials/ansible-galaxy/slides.html:162]: Definitely. What about?

```suggestion
- Run without privileges so if someone gains access they are limited in what they can do.
```

File: topics/contributing/tutorials/slides-with-video/tutorial.md [hunk 3]
```diff
@@ -75,30 +75,41 @@ The captioning process is completely automated, but it means that for very long
...
+
+The same applies to the many terms we read differently from how they are written, e.g. 'src' vs 'source'. Most of us would pronounce it like the latter, even though it isn't spelt that way. Our speaking robot doesn't know what we mean, so we need to spell it out properly.
+
+So we write the definition in the [`bin/ari-map.yml`](https://github.com/galaxyproject/training-material/blob/master/bin/ari-map.yml) file.
+
+### Other Considerations
+
+(*Written 2020-12-16, things may have changed since.*)
+
+Be sure to check the pronunciation of the slides. There are known issues with [Homographs](https://en.wikipedia.org/wiki/Homograph), words spelt the same but pronounced differently often depending on context. Consider "read" for a classic example, or ["analyses"](https://en.wiktionary.org/wiki/analyses#English) for one that comes up often in the GTN. "She analyses data" and "Multiple analyses" are pronounced quite differently based on their usage in sentences. See the [wiktionary](https://en.wiktionary.org/wiki/analyses#English) page for more information, or the [list of English homographs](https://en.wikipedia.org/wiki/List_of_English_homographs) you might want to be aware of.
```
Comment (nsoranzo) [inline topics/contributing/tutorials/slides-with-video/tutorial.md:108]:
```suggestion
Be sure to check the pronunciation of the slides. There are known issues with [heteronyms](https://en.wikipedia.org/wiki/Heteronym_(linguistics)), words spelt the same but having different pronunciation and meaning. Consider "read" for a classic example, or ["analyses"](https://en.wiktionary.org/wiki/analyses#English) for one that comes up often in the GTN. "She analyses data" and "Multiple analyses" are pronounced quite differently based on their usage in sentences. See the [wiktionary](https://en.wiktionary.org/wiki/analyses#English) page for more information, or the [list of English heteronyms](https://en.wiktionary.org/wiki/Category:English_heteronyms) you might want to be aware of.
```

Comment (hexylena) [reply topics/contributing/tutorials/slides-with-video/tutorial.md]: Ah i picked the wrong one from their venn diagram. Cheers.

**Comment by hexylena** [comment]
Ok, I've done as much as I have energy for, before the holidays. Maybe some of the other GAT folks can write some of the other slides.

**Review by martenson — APPROVED** [review]

**Comment by hexylena** [comment]
Thanks a million @martenson @nsoranzo! I'm going to merge this in so I can start working on the video upload machinery (#2220) and we can get everything ready for GAT.

**Comment by hexylena** [comment]
@nsoranzo if you have after the merge :eagle::eyes: feel free to make those comments and i'll get them in another PR (or you can of course).

**Pull request merged**